multiversx_sdk_cli/cli_validators.py

```diff
@@ -182,10 +182,10 @@ def do_stake(args: Any):
             guardian_2fa_code=args.guardian_2fa_code,
         )
     else:
-        validators_file = Path(args.validators_file)
+        validators_file = _load_validators_signers(args.validators_file)
```

Comment:
I would rename this variable.. it is not a validators file

Reply:
renamed it